src/Tribe/Events/Events_Service_Provider.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Tribe\Tickets\Events;
+
+class Events_Service_Provider extends \tad_DI52_ServiceProvider {
```

Comment:
This is already in the `Events` directory and namespace, why not call it just `Service_Provider`?

Reply:
Main.php uses many different Service Providers. I'd have to alias them: `use Tribe\Tickets\Events\Service_Provider as Events_Service_Provider;`

No big deal, but the original issue isn't either. Just a matter of choosing the lesser evil. I'll leave that decision to you, what do you think?

Reply:
I'd rename it `Service_Provider`; classes that need to use it can alias or use the fully qualified name.

Reply:
Done. Thanks. https://github.com/moderntribe/event-tickets/pull/1633/commits/ad0ea4223528efd1003622a578ab4a6d3590ec72